lang/cy.json

```diff
@@ -34,8 +34,8 @@
     "yourDetails": "Eich manylion",
     "firstNames": "Enwau cyntaf",
     "lastName": "Enw olaf",
-    "otherNames": "Unrhyw enwau eraill y cewch eich adnabod wrthynt (dewisol)",
-    "otherNamesLabel": "Unrhyw enwau eraill rydych chi’n eu ddefnyddio",
+    "otherNamesYouAreKnownByOptional": "Unrhyw enwau eraill y cewch eich adnabod wrthynt (dewisol)",
+    "otherNamesYouAreKnownBy": "Unrhyw enwau eraill rydych chi’n eu ddefnyddio",
```

Comment:
I think if you change the key it may need to be put back to "Welsh", as I doubt weblate will allow a nice query for that

Reply:
Ah good point